migrate_nidirect_link/src/EventSubscriber/PostMigrationSubscriber.php

```diff
@@ -67,17 +67,12 @@ public function onMigratePostImport(MigrateImportEvent $event) {
 
       // Retrieve all aliases from D7.
       $query = $conn_migrate->query(
-        "select source, alias from {url_alias}");
+        "select source, alias from {url_alias} where source not in ('node/13638', 'node/13639', 'node/13640', 'node/13641', 'node/13642')");
```

Comment:
There are only five Feature + FCL nodes so while this does duplicate what's in the migrate config, it works.

Reply:
Any chance these will change and if that is the case maybe place these in a vertical array near the top with a text explanation so it can easily be reviewed and edited?

Reply:
Good shout. It's always possible they could change, probably not a huge amount though. The migrate process plugin has a fixed configuration so I'll have to comment accordingly so any changes are reflected in both places.